voluncheer/profiles/forms/organizations.py

```diff
@@ -25,3 +25,16 @@ def save(self, commit=True):
             name=self.cleaned_data.get("name"),
         )
         return user
+
+class OrganizationChangeForm(UserChangeForm):
```

Comment:
Please add a docstring, even if it's a simple one-liner. Here and elsewhere.

https://peps.python.org/pep-0008/#documentation-strings

Reply:
Sure thing! Thanks for the correction